src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidator.java

```diff
@@ -121,7 +121,7 @@ private boolean validatePublicRegisterName(String publicRegisterName, String pub
         var compoundQualifiedFieldName = String.format("%s and %s", qualifiedFieldNamePublicRegisterName, qualifiedFieldNamePublicRegisterJurisdiction);
 
         return StringValidators.isNotBlank(publicRegisterName, qualifiedFieldNamePublicRegisterName, errors, loggingContext)
-                && StringValidators.isLessThanOrEqualToMaxLength(String.format(CONCATENATED_STRING_FORMAT, publicRegisterName, publicRegisterJurisdiction), 160, compoundQualifiedFieldName, errors, loggingContext)
+                && StringValidators.isLessThanOrEqualToMaxLength(String.format(CONCATENATED_STRING_FORMAT, publicRegisterName, publicRegisterJurisdiction), 159, compoundQualifiedFieldName, errors, loggingContext)
```

Comment:
I think the length check should still be 160, as the name and jurisdiction should be max 159, the comma between them makes it 160.
So i believe we just need to change the error message that gets returned to tell the user they can only enter 159 chars. I would check this with Sarah (who raised the bug)

Reply:
Yes, that's correct - the validation message should read that they are only allowed to enter a max of 159 characters.

Reply:
The error message is generated by the validation itself right? Would that require extra work?